src/Microsoft.Health.Fhir.Core/Features/Search/Converters/MoneyNodeToQuantitySearchValueTypeConverter.cs

```diff
@@ -0,0 +1,61 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using EnsureThat;
+using Hl7.Fhir.ElementModel;
+using Hl7.FhirPath;
+using Microsoft.Health.Fhir.Core.Features.Search.SearchValues;
+using Microsoft.Health.Fhir.ValueSets;
+
+namespace Microsoft.Health.Fhir.Core.Features.Search.Converters
+{
+    /// <summary>
+    /// A converter used to convert from <see cref="Money"/> to a list of <see cref="QuantitySearchValue"/>.
+    /// </summary>
+    public class MoneyNodeToQuantitySearchValueTypeConverter : FhirNodeToSearchValueTypeConverter<QuantitySearchValue>
+    {
+        public MoneyNodeToQuantitySearchValueTypeConverter()
+            : base("Money")
+        {
+        }
+
+        protected override IEnumerable<ISearchValue> Convert(ITypedElement value)
+        {
+            var decimalValue = (decimal?)value.Scalar("value");
+
+            if (decimalValue == null)
+            {
+                yield break;
+            }
+
+            var currency = value.Scalar("currency")?.ToString();
+
+            // Currency information is specified differently if we are running STU3.
+            if (currency == null)
+            {
+                var code = value.Scalar("code")?.ToString();
+                var system = value.Scalar("system")?.ToString();
+
+                if (code == null)
+                {
+                    yield break;
+                }
+
+                yield return new QuantitySearchValue(
+                    system,
+                    code,
+                    decimalValue.GetValueOrDefault());
+            }
+            else
+            {
+                yield return new QuantitySearchValue(
+                    CurrencyValues.System,
```

Comment:
@brendankowitz I noticed that in `CodeNodeToTokenSearchValueTypeConverter.cs`, `system` is pulled from the `codeSystemResolver`. That doesn't seem to have "urn:iso:std:iso:4217", though. Is there a nicer way to pull the system info from the POCO instead of creating a new class?

Reply:
These Converters don't use the POCOs. I think we should be aiming to have code that operates more generically but can pull in the metadata that it needs

Reply:
Oops, sorry. By POCO, I meant the `ITypedElement` value that is passed into the `MoneyNodeToQuantitySearchValueTypeConverter`.

While debugging, I can see that "urn:iso:std:iso:4217" gets populated in the `ITypedElement` object. I'm not sure if this is just debugger magic or if there is a way to pull out the value.

![2020-11-17 10_31_19-](https://user-images.githubusercontent.com/54082711/99864557-e7bd6e80-2b58-11eb-80bb-c61dbd807c95.png)

Reply:
As discussed in scrum, Brendan will check with Gino to see if there is a way to pull the "urn:iso:std:iso:4217" system value from FHIR libraries (@brendankowitz please correct me if I misinterpreted this).

Reply:
A bit more context: The value you see there, is from the Poco objects, however, it is also possible to create an ITypedElement using JsonNode, which will not have these values embedded.

I was mainly referring to Gino's codegen as a better way of creating the codesystem mappings over the powershell that currently does it. I think that those values should be there, and if they are not, then I'm interested to find out why not.

Reply:
Thanks, that is helpful 👍 

As discussed, it doesn't make sense to wait for the switch to codegen to get this change in. I added a comment in 4827b96.